.lando.yml

```diff
@@ -35,6 +38,10 @@ services:
     type: mailhog
     hogfrom:
       - appserver
+  node:
```

Comment:
I think we generally don't have a need to run node directly, so I wouldn't add it by default.

Reply:
`npm` tool uses `node` service.

Reply:
Ah, yes, of course that makes sense. I read too quickly and didn’t realize this is a service and not a command.